.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: generic
 
+sudo: false
+dist: trusty
+
```

Comment:
*However*, when I look at the Travis report https://travis-ci.org/MDAnalysis/mdanalysis/jobs/244601325 it says explicitly _ This job ran on our container-based trusty beta. Please read our blog post about the public beta._ so the blog instructions are not very clear.